project/Dependencies.scala

```diff
@@ -114,8 +114,7 @@ object Dependencies {
   val springDeps = Seq(
     jacksonDataFormatProto,
     "org.springframework.boot" % "spring-boot" % SpringVersion,
-    ("org.springframework.boot" % "spring-boot-starter-webflux" % SpringVersion)
-      .exclude("org.springframework.boot", "spring-boot-starter-tomcat"))
```

Comment:
This was previously adding a dependency on `spring-boot-starter-web` and then excluding tomcat. 

At a later point, we changed it to `spring-boot-starter-webflux` because we need access to the reactive WebClient. The change brought in the embedded netty that we see now starting along side akka-http. 

We can't simply exclude the dependency because the WebClient needs it, so we need to disable it by configuring: `spring.main.web-application-type=none`